src/node_crypto.cc

```diff
@@ -5532,6 +5536,64 @@ void ExportChallenge(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(outString);
 }
 
+
+// Convert the input public key to compressed, uncompressed, or hybrid formats.
+void ConvertKey(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  CHECK_EQ(args.Length(), 3);
+
+  THROW_AND_RETURN_IF_NOT_BUFFER(args[0], "Public key");
```

Comment:
Since this is adding a new API, please handle the input type checking in javascript and use the internal/errors mechanism.

Reply:
Done!